Code/Tools/ProjectManager/Source/GemCatalog/GemItemDelegate.cpp

```diff
@@ -431,7 +470,7 @@ namespace O3DE::ProjectManager
         return doc;
     }
 
-    void GemItemDelegate::DrawText(const QString& text, QPainter* painter, const QRect& rect, const QFont& standardFont) const 
+    void GemItemDelegate::DrawText(const QString& text, QPainter* painter, const QRect& rect, const QFont& standardFont, const bool& greyedOut) const 
```

Comment:
I'm starting to wonder why we don't just add a `color` parameter to this function and pass in the color at the call sites instead...

Reply:
per my update in the second comment regarding this, I realized I can remove the colored text drawing for this PR, since it's not being used at all. As such this topic may need to be revisited at a later time.